com.unity.shadergraph/Editor/Generation/TargetResources/StructFields.cs

```diff
@@ -27,8 +27,8 @@ public struct Attributes
                 "COLOR", subscriptOptions : StructFieldOptions.Optional);
             public static FieldDescriptor instanceID = new FieldDescriptor(Attributes.name, "instanceID", "", ShaderValueType.Uint,
                 "INSTANCEID_SEMANTIC", "UNITY_ANY_INSTANCING_ENABLED");
-            public static FieldDescriptor vertexID = new FieldDescriptor(Attributes.name, "vertexID", "ATTRIBUTES_NEED_VERTEXID", ShaderValueType.Uint,
-                "SV_VertexID", subscriptOptions: StructFieldOptions.Optional);
+            public static FieldDescriptor vertexID = new FieldDescriptor(Attributes.name, "vertexID", "", ShaderValueType.Uint,
```

Comment:
Same comment as before, will this still be optional for non dots?

Reply:
We could add hybrid define around it and have it optional for other cases.